iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/azure/AzureClearTextProtocolsCheckPart.java

```diff
@@ -29,8 +32,15 @@
   @Override
   protected void registerResourceConsumer() {
     register("azurerm_spring_cloud_app",
-      resource -> resource.attribute("https_only")
-        .reportIfAbsence(MESSAGE_OMITTING)
-        .reportIfFalse(MESSAGE_CLEAR_TEXT));
+      reportIfAttributeIsAbsentOrFalse("https_only"));
+
+    register(List.of("azurerm_function_app", "azurerm_function_app_slot"),
+      reportIfAttributeIsAbsentOrFalse("https_only"));
```

Comment:
As already discussed for SONARIAC-296, you can combine these consumers. However, you can fix this in your next PR.